DXRando/DeusEx/Classes/Json.uc

```diff
@@ -170,180 +181,273 @@ static function string JsonGetEscapedChar(string c) {
             return Chr(13); //Carriage return
         case "t":
             return Chr(9); //Tab
-        case Chr(34): //Quotes
-        case Chr(92): //Backslash
+        case "\"": //Quotes
+        case "\\": //Backslash
             return c;
         default:
             return "";
     }
 }
 
-static function bool _IsJson(string msg, int length) {
+static function bool _IsJson(string msg) {
```

Comment:
maybe we don't need _IsJson, since JsonStripSpaces would've already returned an empty string in either of these cases

gotta check what `msg = Mid(msg, InStr(msg, "{"));` does when there's no {